Sources/Jaeger/Jaeger/Extensions/TimeInterval+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  TimeInterval+Extensions.swift
+//  Jaeger
+//
+//  Created by Simon-Pierre Roy on 10/29/18.
+//
+
+import Foundation
+
+/// A type to express time intervals in microseconds.
+typealias MicroSeconds = Double
+
+extension TimeInterval {
+    /// A time interval expressed in microseconds.
+    var microseconds: MicroSeconds {
```

Comment:
This'll need a change too:
```suggestion
    var microseconds: Microseconds {
```